pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt

```diff
@@ -48,6 +50,9 @@ import androidx.compose.ui.unit.dp
  *
  * @param pagerState the state object of your [Pager] to be used to observe the list's state.
  * @param modifier the modifier to apply to this layout.
+ * @param size the size of indicators should be displayed, defaults to [PagerState.pageCount].
```

Comment:
would be nice to also mention the use case when it is usually needed, I mean this looping scrollable pager